### DISCUSSION TITLE
Resource or Page

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#5360] [General]
Can I make a Resource or Page without having a Model, that shows a table with data from another 3 models? I Have:

Models:
PaycheckCommon
PaycheckAdm
PaycheckDirectory

with it respective resources to manage, I need one more section that can combine those models, for example show a list o paychecks a person can have. Thanks in advance.

**Comment by danharrin** [comment]
How about a relation manager? https://filamentphp.com/docs/2.x/admin/resources/relation-managers
